roles/openshift_facts/tasks/main.yml

```diff
@@ -1,116 +0,0 @@
----
-- name: Detecting Operating System
-  stat:
-    path: /run/ostree-booted
-  register: ostree_booted
-
-# Locally setup containerized facts for now
-- set_fact:
-    l_is_atomic: "{{ ostree_booted.stat.exists }}"
-- set_fact:
-    l_is_containerized: "{{ (l_is_atomic | bool) or (containerized | default(false) | bool) }}"
-    l_is_openvswitch_system_container: "{{ (openshift_use_openvswitch_system_container | default(openshift_use_system_containers) | bool) }}"
-    l_is_node_system_container: "{{ (openshift_use_node_system_container | default(openshift_use_system_containers) | bool) }}"
-    l_is_master_system_container: "{{ (openshift_use_master_system_container | default(openshift_use_system_containers) | bool) }}"
-    l_is_etcd_system_container: "{{ (openshift_use_etcd_system_container | default(openshift_use_system_containers) | bool) }}"
-- set_fact:
-    l_any_system_container: "{{ l_is_etcd_system_container or l_is_openvswitch_system_container or l_is_node_system_container or l_is_master_system_container }}"
-- set_fact:
-    l_etcd_runtime: "{{ 'runc' if l_is_etcd_system_container else 'docker' if l_is_containerized else 'host' }}"
-
-
-- name: Validate python version
-  fail:
-    msg: |
-      openshift-ansible requires Python 3 for {{ ansible_distribution }};
-      For information on enabling Python 3 with Ansible, see https://docs.ansible.com/ansible/python_3_support.html
-  when:
-  - ansible_distribution == 'Fedora'
-  - ansible_python['version']['major'] != 3
-  - r_openshift_facts_ran is not defined
-
-- name: Validate python version
-  fail:
-    msg: "openshift-ansible requires Python 2 for {{ ansible_distribution }}"
-  when:
-  - ansible_distribution != 'Fedora'
-  - ansible_python['version']['major'] != 2
-  - r_openshift_facts_ran is not defined
-
-# Fail as early as possible if Atomic and old version of Docker
-- block:
-
-  # See https://access.redhat.com/articles/2317361
-  # and https://github.com/ansible/ansible/issues/15892
-  # NOTE: the "'s can not be removed at this level else the docker command will fail
-  # NOTE: When ansible >2.2.1.x is used this can be updated per
-  # https://github.com/openshift/openshift-ansible/pull/3475#discussion_r103525121
-  - name: Determine Atomic Host Docker Version
-    shell: 'CURLY="{"; docker version --format "$CURLY{json .Server.Version}}"'
-    register: l_atomic_docker_version
-
-  - assert:
-      msg: Installation on Atomic Host requires Docker 1.12 or later. Please upgrade and restart the Atomic Host.
-      that:
-      - l_atomic_docker_version.stdout | replace('"', '') | version_compare('1.12','>=')
-
-  when:
-  - l_is_atomic | bool
-  - r_openshift_facts_ran is not defined
-
-- name: Load variables
```

Comment:
This task didn't make it over to the new file.  Therefore the logic for Fedora is missing and will not install the proper `required_packages`.